bpf/lib/icmp6.h

```diff
@@ -30,8 +30,10 @@
 #define ICMP6_MULT_RA_MSG_TYPE		151
 #define ICMP6_MULT_RT_MSG_TYPE		153
 
-/* If no specific action is specified, drop unknown neighbour solicitation
- * messages.
+#define SKIP_HOST_FIREWALL	-1
+
+/* If not specific action is specified, drop unknown neighbour solication
+ * messages
```

Comment:
Also lost from a rebase:
```suggestion
/* If no specific action is specified, drop unknown neighbour solicitation
 * messages.
```